woocommerce/woocommerce-ios
# [Mobile Payments] Remove html tags from receipt generation

Closes #4137

## Description:

### The PR removes HTML symbols from:
* Receipt item title
* Application preferred name
* Dedicated file name

### Adds a unit test on `ReceiptRenderer`

## To test:
* Modify the code so one or all of the fields mentioned above have HTML tags (check `ReceiptRenderer`)
* Make a payment or open stored receipt
* Notice that the modification doesn't affect the generated receipt


@ctarda please try to point on anything you see that doesn't look good, even if it's nitpicks - should be good for educational purposes =)
